### DISCUSSION TITLE
Building on Windows

### DISCUSSION THREAD

**Discussion by DavidRueter** [pymssql/pymssql#739] [General]
I previously have been able to build pymssql in a Windows development environment, including building openssl and freetds from source, as per the instructions in [the docs](https://github.com/pymssql/pymssql/blob/master/docs/building_and_developing.rst)

Are these instructions correct and current?  I would like to freshen up my development environment and resume building pymssql and dependencies from source but would first like to know if there are any changes or updates to the build workflow.

**Comment by termim** [comment]
There is a script [build.py](https://github.com/pymssql/pymssql/blob/master/dev/build.py) that is used to build `pymssql` wheels.
On windows it downloads and builds FreeTDS and iconv and then builds the wheel.
You can check how it is used in github [actions](https://github.com/pymssql/pymssql/blob/f486f7d79bd2583eba96323cdeca34eb8f14cd2a/.github/workflows/test_windows.yml#L48).
